fixture/hr_administration.py

```diff
@@ -0,0 +1,32 @@
+from fixture.step import StepHelper
+from selenium.webdriver.remote.webdriver import WebDriver
+
+class HrAdministration:
+    add_user_button = "//div[@id='systemUserDiv'] //*[text()='add']"
+    filter_users_button = '//a[@data-tooltip="Filter"]/i'
+    filtered_username = '//span[text()="admin"]'
```

Comment:
this will work only if you set 'admin' to filter, please change it to: '(//div[@class="list-container"] //tr //td)[2]'